docs/source/module_configuration/serverless.rst

```diff
@@ -134,3 +134,15 @@ directory is pre-compiled) via the ``skip_npm_ci`` module option:
           - path: myslsproject.sls
             options:
               skip_npm_ci: true
+
+Specifying serverless config file
+---------------------------------
+Name of your configuration file, if other than serverless.yml|.yaml|.js|.json.
+::
+
+    ---
+    deployments:
+        - modules:
+            - path: sampleapp.sls
+              options:
+                config: sampleapp.yml
```

Comment:
(totally my preference here -- I think it makes sense that you started with the identical option name that sls uses) -- can we make the option `configfile` instead? I'd like to to be more explicit.

Reply:
I'll change it, thanks for reviewing!

Reply:
or maybe even `sls_file` to reduce our use of the word *config*?

Reply:
what about sls_configfile?, I'll update the PR with sls_configfile for the moment. Just ping me if there's any concern